tr.org.pardus.pen.yaml

```diff
@@ -0,0 +1,23 @@
+app-id: tr.org.pardus.pen
+runtime: org.kde.Platform
+runtime-version: 5.15-23.08
+sdk: org.kde.Sdk
+command: /app/usr/bin/pardus-pen
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=wayland
+  - --device=dri
+  - --share=network
+  - --filesystem=host
+modules:
+  - name: pardus-pen
+    buildsystem: simple
+    build-commands:
+      - qmake
+      - make
+      - make install DESTDIR=${FLATPAK_DEST} INSTALL_ROOT=${FLATPAK_DEST}
```

Comment:
`$FLATPAK_DEST` is NOT a `DESTDIR`. It's a `PREFIX`. here `DESTDIR` is empty (unset)

Reply:
but if you use buildsysten qmake then you don't even need this.